### PR TITLE
Check goconst output

### LINKED ISSUE #324: goconst doesn't fail the CI if errors were found
https://travis-ci.org/github/RedHatInsights/insights-results-aggregator/jobs/664002405?utm_medium=notification&utm_source=github_status

### PR DESCRIPTION
**Pull request by tisnik**
# Description

Check `goconst` output.

Fixes #324

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## Testing steps

Let's see on CI how it works (worked locally for `./goconst.sh`)

### CODE REVIEW

File: goconst.sh [hunk 1]
```diff
@@ -15,4 +15,12 @@
 
 
 GO111MODULE=off go get github.com/jgautheron/goconst/cmd/goconst
-goconst -min-occurrences=3 ./...
+
+if [[ $(goconst -min-occurrences=3 ./... | tee /dev/tty | wc -l) -ne 0 ]]
```
Comment (Sergey1011010) [inline goconst.sh]: what's the purpose of `tee /dev/tty` here?

Comment (tisnik) [reply goconst.sh]: I'd like to see the actual output from `goconst`

Comment (tisnik) [reply goconst.sh]: this is semi-standard way how to do it

Comment (Sergey1011010) [reply goconst.sh]: ah, got it, thanks, then looks good to me

**Pull request merged**